proto/sandbox.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+import "types.proto";
+
+option go_package = "proto";
+
+service Sandbox {
+    // Runs new Firecracker VM instance
+    rpc CreateVM(CreateVMRequest) returns (CreateVMResponse);
+
+    // Stops existing Firecracker instance by VM ID
+    rpc StopVM(StopVMRequest) returns (google.protobuf.Empty);
+
+    // Gets VM instance socket file location
+    rpc GetVMAddress(GetVMAddressRequest) returns (GetVMAddressResponse);
+}
+
+// CreateVMRequest specifies creation parameters for a new FC instance
+message CreateVMRequest {
```

Comment:
(This is just a general question, not something that should probably be addressed in this CR)

Do you think there's any possibility of sharing the struct definitions here with the similar ones used in the firecracker-go-sdk? On the surface, it seems like there's a lot of overlap.

Just sharing definitions would probably be beneficial, but even better would be if we could just take the firecracker-go-sdk and point it at fc-control and start a VM up.

The main issue I can think of would be that the go-sdk doesn't currently have a notion of VMID, though I wonder if it would be much effort to add a layer there that incorporates it in a way such that it's compatible with both the firecracker socket and fc-control. Maybe that's easier said than done though, just curious what your thoughts are.

Reply:
There is an overlap and this is why we have to keep [taskfirecrackeropts.go](https://github.com/firecracker-microvm/firecracker-containerd/blob/master/runtime/taskfirecrackeropts.go) to convert from one to another (not super happy with it either). SDK generates models from swagger file (pulled from FC repo), while here we generate from protobuf definition in order to use GRPC. 
I definitely be happy to avoid this overlap. We might want to consider generating protobuf from swagger as well, so we don't need to duplicate lots of contracts as one possible solution.

Reply:
Another another option: handle it similarly to how `containerd` handles OCI specs: [1](https://github.com/containerd/containerd/blob/932f883e49af925380a3fdce6b8db189915f93fb/containers/containers.go#L56) and [2](https://github.com/containerd/containerd/blob/932f883e49af925380a3fdce6b8db189915f93fb/container_opts.go#L216) (e.g. pack sdk's structs to `type.Any`)